components/calendar/calendar.js

```diff
@@ -625,6 +633,10 @@ class Calendar extends LocalizeCoreElement(RtlMixin(LitElement)) {
 		}
 	}
 
+	getShownValue() {
```

Comment:
I was a little torn on what to return here... a JavaScript `Date` or an ISO date string. I ultimately chose the string to be consistent with the properties. I suspect the properties use this format because it's easier for the consumer to set their HTML attributes as strings. But consumers might prefer the `Date` object and providing strings forces them to convert it themselves.

Anyway, I'm open to using either format here, or renaming this method if you can think of a better name.